frontend/env_config/.env.dev

```diff
@@ -2,4 +2,5 @@
 VITE_APP_TITLE=HN Web (Dev)
 VITE_KEYCLOAK_URL=https://common-logon-dev.hlth.gov.bc.ca/auth/
 VITE_SITEMINDER_LOGOUT=https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://mspdirect-dev.apps.silver.devops.gov.bc.ca
-VITE_SERVICE_URL=https://mspdirect-dev-api.apps.silver.devops.gov.bc.ca/
\ No newline at end of file
+VITE_SERVICE_URL=https://mspdirect-dev-api.apps.silver.devops.gov.bc.ca/
+VITE_KEYCLOAK_CLIENT_ID=MSPDIRECT-WEB
```

Comment:
This parameter also needs to be removed from .env if we are making it environment specific.

Reply:
Updated